M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -241,7 +241,9 @@ randomSubSimplicialComplex(AbstractSimplicialComplex) := AbstractSimplicialCompl
 ambientAbstractSimplicialComplexSize = method() 
 
 ambientAbstractSimplicialComplexSize(AbstractSimplicialComplex) := (K) -> (
-    max flatten(K_0))
+    L := flatten(K_0);
+    if L != { } then return(max L) else return(0) -- by default {{}} has size 0
```

Comment:
Thanks for the updates!

I'll go ahead and merge this, but a couple stylistic things for future reference:

* It isn't necessary to use `return` in the last line of a function.
* Parentheses aren't necessary for functions with a single argument.

See also the [Package Writing Style Guide](https://github.com/Macaulay2/M2/wiki/Package-Writing-Style-Guide) for these and lots of other tips.

So for example, this could be simplified to:

```m2
L := flatten K_0;
if L != { } then max L else 0
```
or even more compactly:
```m2
if (L := flatten K_0) != {} then max L else 0
```